data/avro-schema/src/main/resources/avro/net/corda/data/crypto/wire/ops/rpc/queries/ByIdsRpcQuery.avsc

```diff
@@ -2,15 +2,15 @@
   "type": "record",
   "name": "ByIdsRpcQuery",
   "namespace": "net.corda.data.crypto.wire.ops.rpc.queries",
-  "doc": "Request to lookup for keys by their ids, the maximum number of items is 20.",
+  "doc": "Request to lookup for keys by their ids. Ids can be either short or full key ids. The maximum number of items is 20",
```

Comment:
why the limit of 20? Does the backend code break up large requests into a series of size 20 batch requests?

Reply:
`SigningKeyStoreImpl` look up methods throw if requested keys found to be in number more than 20 (this was already there).

Reply:
This number 20 (and the version inByIdsFlowQuery) feels like a server limitation rather than something which should be in the interface.

But it's already there, we should highlight in both places as you are doing if we we are to mention it one place, and I wouldn't hold up approval on this.